src/components/TvDetails/index.tsx

```diff
@@ -80,6 +80,7 @@ const messages = defineMessages({
   seasons: '{seasonCount, plural, one {# Season} other {# Seasons}}',
   episodeRuntime: 'Episode Runtime',
   episodeRuntimeMinutes: '{runtime} minutes',
+  streamingproviders: 'Currently Streaming On',
```

Comment:
I feel like “Streaming On” would be more concise? Especially if it’s an upcoming series/movie that’s not streaming yet.

Reply:
Perhaps even “Stream Providers” or “Streaming Services” instead?

Reply:
This actually lists the _current_ streaming providers only, so I wanted to emphasize that.  Upcoming items wouldn't have any streaming providers to list.

Reply:
> Perhaps even “Stream Providers” or “Streaming Services” instead?

I originally had `Streaming Providers`, but it felt somewhat awkward so I settled on this.

Reply:
Cool 👍